src/components/Dashboard/FilingHistoryList/StaffFiling.vue

```diff
@@ -1,29 +1,31 @@
 <template>
```

Comment:
I updated this component because I was going to reuse it for the Consent to Continuation Out ledger item. Then I decided not to reuse it, but I kept the cleanup/updates.

Sample court order (looks the same as before):

![image](https://user-images.githubusercontent.com/10002889/216710955-135de3fe-7e2d-45c4-8285-e4b7486102be.png)